### DISCUSSION TITLE
QR Code interface/display

### DISCUSSION THREAD

**Discussion by Nitwel** [directus/directus#7658] [Feature Requests]
It could be useful for companies to generate QR Codes for items.

My thoughts on this are:
1. We could make it similar to button-links that take a field-template and generates a special QR Code for each item. (`alias`)
2. If you want the QR Code to be persistent/access it in the frontend, we could make this into a interface for images. So when you change an item, the interface regenerates its image and when you save it, updates the image file. This would allow some cool stuff like previewing the QR Code inside the cards view. (`image`)
3. Have a QR Code interface/display for `string/text` that generates an QR Code of of that special text.

**Comment by benhaynes** [comment]
Happy to mock-up a design for this! Just need to know all the requirements for the UI.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
